tests/test_sandbox/test_sandbox_lifecycle.py

```
@@ -173,6 +173,47 @@ async def test_apply_cmd_executed(self):
         assert any("git apply" in cmd for cmd in verify_sb._exec_log)
         await lc.teardown()
 
+    @pytest.mark.asyncio
+    async def test_workspace_presence_check_resolves_efs_session_path(self):
+        """ws_present check resolves the EFS session tar path alongside the default."""
+        ctx, mgr = self._make_ctx()
```

Ruff (0.16.0)

[warning] Unpacked variable `mgr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)